.gitignore

```diff
@@ -32,3 +32,5 @@
 /config/bbgo.yaml
 
 /pkg/server/assets.go
+
+*.env
```

Comment:
not `.env.*` ?

Reply:
see this https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use

Reply:
Oh! It's my personal usage convention.. sorry

Reply:
should be `.env.*` here